src/ztest_sfpf.sfpf.xml

```diff
@@ -19,7 +19,7 @@
      <TEXT>test form</TEXT>
     </FPCONTEXTT>
    </CONTEXTT>
-   <LAYOUT href="#o102"/>
+   <LAYOUT href="#o1"/>
    <LAYOUTT>
     <FPLAYOUTT>
      <NAME/>
```

Comment:
the LAYOUT part is the thing that is now in the xdp file, so I would expect this to be removed from the xml?

Reply:
A) As far as I know, it is not possible to remove layout form a form - at least, there is no API for it. I tried to set the layout to empty string but it breaks serialization:
https://github.com/larshp/abapGit/pull/2555#discussion_r267453104

B) As far as I know, the element layout consists of XDPs in the form's language and also translated layouts. The pull request I created for abapGit moves the form's layout to the new file, so I believe it is not possible to remove get rid of the element LAYOUT completely without create separate files for all languages.

Reply:
hmm, but do you agree that currently data is duplicated, some part of the LAYOUT binary string is dupliated to the XDP file?

Reply:
Yes, I knew there was redundancy but the systems where I was testing my changes were behaving strange and randomly showing changes, randomly raising exceptions, randomly putting layouts for additional languages.

One have to close sapgui and start it again to make sure the systems behaves as expected.